lib/core/Theme_data/app_text_theme.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class AppTextTheme {
+  static const mobileTextTheme = TextTheme(
+    headlineMedium: TextStyle(fontSize: 18, fontWeight: FontWeight.normal,),
```

Comment:
**suggestion**: create a base text style and then do copywith for all so you can avoid creating infinite text styles.

  static final baseTextStyle = TextStyle(
    fontFamily: FontFamily.signifyRelative,
    fontWeight: AppFontWeight.book,
    color: _defaultTextColor,
    letterSpacing: 0,
  );